meetings/submissions/serializers.py

```diff
@@ -41,3 +45,12 @@ def get_can_edit(self, obj):
         request = self.context.get('request')
         user = request.user
         return (user == obj.contributor or user == obj.conference.admin)
+
+    def get_download_count(self, obj):
+        conn = httplib.HTTPConnection("api.osf.io")
+        #file_id = Submission.objects.filter() <-- Edit this to find the actual fileid
+        conn.request("HEAD","/v2/files/") #not done needs to be vs/files/{fileid}
+        res = conn.getresponse()
+        # figure out response format and return download count
```

Comment:
You should be able to figure out the format for a file response by looking at OSF code or sending requests to the API with postman.